docs/reference/modules/node.asciidoc

```diff
@@ -36,7 +36,18 @@ set `node.roles`, the node is assigned the following roles:
 [IMPORTANT]
 ====
 If you set `node.roles`, ensure you specify every node role your cluster needs.
-Some {stack} features require specific node roles:
+Every cluster requires the following node roles:
+
+* `master`
+* {blank}
++
+--
+`data_content` and `data_hot` +
+OR +
+`data`
+--
```

Comment:
Technically, you can start a cluster without these data roles. However, they're required to create new data streams or indices.